README.md

```diff
@@ -47,6 +47,7 @@ If you're using `gopkg.in`, you can still rely in the `v0` without worrying abou
 - Custom output color space (RGB, grayscale...)
 - Format conversion (with additional quality/compression settings)
 - EXIF metadata (size, alpha channel, profile, orientation...)
+- Trim
```

Comment:
Given that the feature is not GA in `libvips` today, I think it would be convenient adding a note here that the feature is only available in `libvips >= 8.6`.